src/main/java/hudson/plugins/jira/UpdaterIssueSelector.java

```diff
@@ -27,6 +27,6 @@
      * @param listener Current's run listener.
      * @return Set of ids of issues which should be updated.
      */
-    public abstract Set<String> findIssueIds(@Nonnull Run<?, ?> run, @Nonnull JiraSite site, @Nonnull TaskListener listener);
+    public abstract Set<String> findIssueIds(@Nonnull Run<?, ?> run, @Nonnull Updater updater, @Nonnull JiraSite site, @Nonnull TaskListener listener);
```

Comment:
I prefer not change this public method, but default implementation used Updater way that requires refactoring if we want to keep this api unchanged. I try to think about it.